packages/kbn-expandable-flyout/src/index.tsx

```diff
@@ -67,15 +70,18 @@ export const ExpandableFlyout: React.FC<ExpandableFlyoutProps> = ({
     return <></>;
   }
 
-  const width: number = (leftSection?.width ?? 0) + (rightSection?.width ?? 0);
+  const flyoutWidth: string = leftSection && rightSection ? 'l' : 's';
```

Comment:
Clever! I wonder if it's worth bringing up the option to collapse the right section for more real estate in the left section if it ever becomes necessary. Similar to how we can collapse the left side nav in kibana. That may come up with whatever is decided around the `Take Action` flyouts. Either way, I like this, nice work!

Reply:
I did think about that a while back, but it never came up in the  discussions with PMs or UIUX, so I didn't pursue. I can bring it up again. I feel though that right now, the current flow would always have the right section displayed...

Reply:
Oh yea, it's not urgent to the current view, but I could 1000% see a user asking for this in the future

Reply:
by then we'll have the detail page anyway 🤣 😉

Reply:
...or a full screen details flyout 👀 😂